_config.yml

```diff
@@ -25,3 +25,12 @@ aux_links:
   "GitHub":
     - "https://github.com/microsoft/onnxruntime"
 ga_tracking: UA-156955408-1    
+
+# Footer "Edit this page on GitHub" link text
+gh_edit_link: true # show or hide edit this page link
+gh_edit_link_text: "Edit this page on GitHub"
+gh_edit_repository: "https://github.com/microsoft/onnxruntime" # the github URL for your repo
+gh_edit_branch: "gh-pages" # the branch that your docs is served from
+# gh_edit_source: docs # the source that your files originate from
+gh_edit_view_mode: "tree" # "tree" or "edit" if you want the user to jump into the editor immediately
+footer_content: "For documentation questions, please <a href=\"https://github.com/microsoft/onnxruntime/issues/new?labels=component:documentation&title=[Documentation%20Request]\" target=\"_blank\">file an issue</a>."
```

Comment:
Looks like **footer_content** is deprecated https://github.com/pmarsceill/just-the-docs/blob/master/docs/configuration.md#footer-content

Reply:
We can add a footer.html template: https://jekyllrb.com/docs/includes/

Reply:
Updated to use the _includes directory.

Reply:
change footer content to get the body populated with current page info:


{%- assign url = page.url -%}

<p class="text-small text-grey-dk-100 mb-0">For documentation questions, please <a href="https://github.com/microsoft/onnxruntime/issues/new?labels=component:documentation&title=[Documentation%20Request]&body=%5BEnter%20feedback%20here%5D%0A%0A---%0A%23%23%23%23%20Document%20Details%0A*%20Title%3A%20{{ page.title | url_encode }}%0A*%20Page%3A%20%5B{{ url | absolute_url | url_encode }}%5D({{ url | absolute_url | url_encode }})%0A*%20Page%20Source%3A%20{{ page.name | url_encode }}" target="_blank">file an issue</a></p>
© 2021 GitHub, Inc.